openshift/templates/django-postgresql.json

```diff
@@ -328,18 +324,12 @@
       "from": "[a-zA-Z0-9]{16}"
     },
     {
-      "name": "APP_MODULE",
-      "description": "Python dotted path to your Django WSGI application",
-      "value": "project.wsgi"
-    },
-    {
       "name": "APP_CONFIG",
-      "description": "Relative path to Gunicorn configuration file (optional)",
-      "value": "gunicorn_conf.py"
+      "description": "Relative path to Gunicorn configuration file (optional)"
```

Comment:
@jcantrill fyi here's a parameter definition with no value and no generator.  so i think we made the right call in treating it as acceptable.